examples/k8s-upgrade/kubernetes1.5.3-hybrid.json

```diff
@@ -0,0 +1,47 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorVersion": "1.5.3"
+    },
+    "masterProfile": {

```

Comment:
Upgrade is supported for both manageddisks and storageaccount storageProfile types for masters. 

Managed disk upgrade support was added in this PR: https://github.com/Azure/acs-engine/issues/1027

My recommendation would be convert one of these tests to be: 
- Set MasterProfile on one test: "storageProfile": "ManagedDisks" 
- and others can be "storageProfile": "StorageAccount"

This way the major upgrade scenarios will be covered.